kotlinx-coroutines-core/api/kotlinx-coroutines-core.api

```diff
@@ -888,6 +888,16 @@ public final class kotlinx/coroutines/debug/internal/DebugCoroutineInfo {
 	public final fun lastObservedStackTrace ()Ljava/util/List;
 }
 
+public final class kotlinx/coroutines/debug/internal/DebugProbesImpl {
```

Comment:
Have you inferred it by looking at IDEA's code or just by looking at the comments?